R/int_pctl.R

```diff
@@ -34,11 +34,10 @@
 #' The defaults reflect the fewest samples that should be used.
 #'
 #' The computations for each configuration can be extensive. To increase
-#' computational efficiency parallel processing can be used. The \pkg{foreach}
+#' computational efficiency parallel processing can be used. The \pkg{future}
```

Comment:
I would say we opt to document only the recommended approach to prevent new users from getting situated with foreach. If we had more involved setup instructions, I might keep both around to help folks that are in the middle of transitioning, but since they're mostly just references to others' docs I'd say we just point to future.

Leaving this as unresolved in case you want to overrule. :)

Reply:
What about we compromise and only keep in the vignette? since we have linked to that from other places as "this is how you use parallelism with foreach" on StackOverflow and stuff?

with big caviat that {future} is the future?

Reply:
I would switch to only mentioning {future} in the function documentation, possibly with a link to the vignette, because if function documentation is too long, people tend to not read it.

In a vignette, people are looking for more detailed info (IMO) and there is space to provide it, so I'm in favor of keeping a reference there to all that works (= including foreach), especially given that this change is not going to happen overnight. With the clear direction to use {future} in future.

Summary: +1 to Emil!

Reply:
Thanks, yall :)